src/views/networkpolicies/list/NetworkPolicyPage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom-v5-compat';
+
+import { modelToRef, NetworkPolicyModel } from '@kubevirt-ui/kubevirt-api/console';
+import { Tab, Tabs, TabTitleText } from '@patternfly/react-core';
+import { useLastNamespacePath } from '@utils/hooks/useLastNamespacePath';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { MultiNetworkPolicyModel } from '@utils/models';
+
+import MultiNetworkPolicyList from './MultiNetworkPolicyList';
+import NetworkPolicyList from './NetworkPolicyList';
+
+export type NetworkPolicyPageNavProps = {
+  namespace: string;
+};
+
+export const trimLastHistoryPath = (pathName: Location['pathname']): string => {
+  return pathName.endsWith('checkups') ? pathName : pathName.replace(/\/[^\/]*$/, '');
```

Comment:
checkups ?

Reply:
i don't really use this function. Deleted.

Reply:
this function for refresh, to stay in same tab, check kif its working without it

Reply:
yes works on refresh